src/layout/footer/__fixtures__.js

```diff
@@ -0,0 +1,63 @@
+import Faker from "faker"
```

Comment:
I've been following a different convention for these files, but I haven't shared that yet, so it's fine to stay for now.